Sources/Mugen/MugenNotificationCenter.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 import UserNotifications
 import CoreLocation
 
-public struct MugenNotificationCenter {
+public struct MugenNotificationCenter: @unchecked Sendable {
```

Comment:
`MugenNotificationCenter` needs to be thread-safe. Please add appropriate thread-safety mechanisms, such as locking when writing to NotificationStorage, to ensure that MugenNotificationCenter is thread-safe.